lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -220,13 +221,15 @@ class CategoryRenderer {
 
   /**
    * @param {Array<Element>} elements
+   * @param {boolean} expandByDefault
    * @return {Element}
    */
-  renderPassedAuditsSection(elements) {
+  renderPassedAuditsSection(elements, expandByDefault) {
```

Comment:
We've learned over time that every `boolean` we add to a function method, we regret eventually :)

let's do an object instead `const {expandByDefault} = displayOptions`

Reply:
Ah, ok, thanks for the insight.